extensions/identity-hub/src/test/java/org/eclipse/dataspaceconnector/identityhub/processor/CollectionsWriteProcessorTest.java

```diff
@@ -50,17 +51,20 @@ void setUp() {
     void writeCredentials() throws Exception {
         // Arrange
         var credentialId = FAKER.internet().uuid();
-        var verifiableCredentialMap = Map.of("id", credentialId);
-        var data = OBJECT_MAPPER.writeValueAsString(verifiableCredentialMap).getBytes(StandardCharsets.UTF_8);
+        var issuer = FAKER.internet().url();
+        var verifiableCredential = VerifiableCredential.Builder.newInstance()
```

Comment:
It would be nice to extract the code for generating a VerifiableCredential for tests in a common util class in a similar way as: https://github.com/agera-edc/IdentityHub/pull/28/commits/9dbf5f1b80b0ce422dee8d60aff9964ce2b9c267